Mathlib/Order/RelClasses.lean

```diff
@@ -667,109 +637,80 @@ end SubsetSsubset
 /-! ### Conversion of bundled order typeclasses to unbundled relation typeclasses -/
 
 
+@[to_dual instIsReflGe]
```

Comment:
Not for this PR, but it does seem kind of annoying that these instance names all have to be specified.